app/src/main/res/layout/activity_main.xml

```diff
@@ -60,6 +60,17 @@
                 android:layout_height="44dp"
                 android:hint="@string/add_list_hint"
                 />
+
+            <com.joanzapata.iconify.widget.IconTextView
+                android:id="@+id/voice_entry_icon"
+                android:text="@string/voice_icon"
+                android:layout_gravity="end|center_vertical"
+                style="@style/list_item_button"
+                android:layout_marginEnd="60dp"
```

Comment:
This should be 55dp I think so there isn't a gap between the icon space and the add button.